### PR TITLE
fix: endpoints implementation in corda plugin

### LINKED ISSUE #1346: fix (cactus-plugin-ledger-connector-corda): add endpoints implementation
**Describe the bug**

We need to be able to call corda endpoints.

**To Reproduce**

To reproduce this bug, we must call corda endpoints hosted in an express server instead of call directly to corda container. We will receive an error with status 404.

**Expected behavior**

We need that these endpoints return us the same result as if we call directly to corda container.

**Hyperledger Cactus Plugins/Connectors Used**

 - Corda

### PR DESCRIPTION
**Pull request by elenaizaguirre**
Closes #1346

Signed-off-by: Elena Izaguirre <e.izaguirre.equiza@accenture.com>

### CODE REVIEW

**Comment by elenaizaguirre** [comment]
@petermetz Can you take a look at this, please? I am not sure if this is that you wanted me to do. I started with a remote adapter like plugin-keychain-vault-remote-adapter.ts but later I saw that it was simpler to use those endpoints like proxies to container endpoints (as Vault.client in plugin-keychain-vault.ts) because Corda endpoints are not used when we call container endpoints.

File: packages/cactus-plugin-ledger-connector-corda/src/main/typescript/web-services/deploy-contract-jars-endpoint.ts [hunk 1]
```diff
@@ -140,75 +142,12 @@ export class DeployContractJarsEndpoint implements IWebServiceEndpoint {
     }
   }
 
-  private async doDeploy(
```
Comment (elenaizaguirre) [inline packages/cactus-plugin-ledger-connector-corda/src/main/typescript/web-services/deploy-contract-jars-endpoint.ts:143]: @petermetz I changed this code for a call to the container endpoint, do you prefer me to revert to the previous function?

**Review by takeutak — APPROVED** [review]

File: packages/cactus-plugin-ledger-connector-corda/src/main/typescript/web-services/deploy-contract-jars-endpoint.ts (hunk 1 above)
Comment (petermetz) [reply packages/cactus-plugin-ledger-connector-corda/src/main/typescript/web-services/deploy-contract-jars-endpoint.ts:143]: @elenaizaguirre No, it's the right call to keep things in the JVM implementation as long as that works.

**Review by petermetz — CHANGES_REQUESTED** [review]
@elenaizaguirre A few questions:

1. Is the remote adapter pattern not implemented in here? I couldn't find a class for it in the diff.
2. Can the OpenAPI validation be done in the JVM implementation? Right now the validation only happens (I think) if you proxied the requests through the NodeJS app but if you were to hit the JVM app directly it wouldn't perform the validations.

**Comment by elenaizaguirre** [comment]
> @elenaizaguirre A few questions:
> 
> 1. Is the remote adapter pattern not implemented in here? I couldn't find a class for it in the diff.
> 2. Can the OpenAPI validation be done in the JVM implementation? Right now the validation only happens (I think) if you proxied the requests through the NodeJS app but if you were to hit the JVM app directly it wouldn't perform the validations.

@petermetz 
That pattern is not implemented because I used a proxy to access the internal container.
Validations will be directly in the JVM app

**Review by petermetz — APPROVED** [review]
@elenaizaguirre 

> That pattern is not implemented because I used a proxy to access the internal container.

Okay, that's fine for now but we'll have to move that to a remote adapter once we are adding support to Corda 5.x because that one will have support for RPC directly from JS code

**Pull request merged**